libs/core/algorithms/CMakeLists.txt

```diff
@@ -225,6 +226,7 @@ set(algorithms_compat_headers
     hpx/algorithms.hpp => hpx/algorithm.hpp
     hpx/parallel/traits/projected.hpp => hpx/algorithms/traits/projected.hpp
     hpx/parallel/traits/projected_range.hpp => hpx/algorithms/traits/projected_range.hpp
+    hpx/traits/is_relocatable.hpp => hpx/algorithms/traits/is_relocatable.hpp
```

Comment:
Yu don't need this entry. This is only for compatibility with old locations of headers that have been moved around.